R/data-set.R

```diff
@@ -125,33 +146,57 @@ DataSet <- R6::R6Class(
       }
       private$throwPropertyIsReadonly("yErrorValues")
     },
+
     #' @field molWeight Molecular weight of the yValues in g/mol
+
     molWeight = function(value) {
       if (missing(value)) {
         molWeight <- private$.yColumn$molWeight
         if (is.null(molWeight)) {
           return(NULL)
         }
-        return(toUnit(quantityOrDimension = ospDimensions$`Molecular weight`, values = molWeight, targetUnit = ospUnits$`Molecular weight`$`g/mol`))
+        return(toUnit(

```

Comment:
This is better like this

Reply:
I have reverted this change.

Reply:
Well one of this was good lol.
;-)

Reply:
LOL, sorry misunderstood.

I think we need to have a coding standard for this, instead of me changing the formatting haphazardly. 

Specifically, at what point we prefer to have one argument per line: when it is over 80 characters?, when there are more than 3 arguments? etc.

Reply:
I would love the styler to do it for us. Personal preferences is irrelevant. As long as a machine formats the  code. Can we have the styler do if for us?

Reply:
Do it*

Reply:
> I would love the styler to do it for us

There has been [this](https://github.com/r-lib/styler/issues/247) issue pending since 4 years and yet to be resolved. If it is ever resolved, it might do this for us.